index.html

```diff
@@ -40,6 +40,187 @@
     </style>
   </head>
   <body>
+
+    <header>
+     <img src="2291219-200.png" alt="logo of a potted vine">
+     <h2>Happy Plant Gardening</h2>
+     <nav>
+      <h3>Site Navigation</h3>
+      <ul>
+        <li><a href=#>Home</a></li>
+        <li><a href=#>About</a></li>
+        <li><a href=#>Contact</a></li>
+      </ul>
+     </nav>
+    </header>
+
+    <main>
+      <section>
+      <h1>Caring for Orchids While in Bloom</h1>
+        <img src="photo-1586799958784-8687bc5d659f.avif" alt="purple orchid">
+        <p><i>(This page explains caring for orchids while in bloom. If your orchid has finished flowering, or you require more in depth potting instructions, please click <a href=#>here</a>.)</i></p>
+      </section>
```

Comment:
Not technically wrong, so take this with a grain of salt, but you don't need to put a section tag around your h1.